services/aur_cache/main.go

```diff
@@ -65,6 +65,11 @@ func info(cache store.Cache) http.HandlerFunc {
 
 		entry, err := cache.GetInfoEntry(name)
 		if err == nil {
+			if !entry.OnAur {
+				http.Error(w, "no results", http.StatusNoContent)
```

Comment:
StatusNoContent isn't really an error (and shouldn't have content)

Reply:
How about a 404? That one would make sense to me